run.py

```diff
@@ -336,6 +339,59 @@ def multi_synthesis(queries: List[AudioQuery], speaker: int):
 
         return FileResponse(f.name, media_type="application/zip")
 
+    @app.post(
+        "/connect_waves",
+        response_class=FileResponse,
+        responses={
+            200: {
+                "content": {
+                    "audio/wav": {"schema": {"type": "string", "format": "binary"}}
+                },
+            }
+        },
+        tags=["その他"],
+        summary="複数の音声を一つに結合する",
+    )
+    def connect_waves(waves: List[str]):
+        """
+        base64エンコードされたwavデータを一纏めにし、wavファイルで返します。
+        """
```

Comment:
`str`だとこのAPIに何の文字列を渡せばいいか利用者に伝わりづらそうなので、base64エンコードされた音声ファイルであることがわかるようなsummaryにするのはどうでしょう。

例えばこんな感じで：
```suggestion
        summary="base64エンコードされた複数のwavデータを一つに結合する",
    )
    def connect_waves(waves: List[str]):
        """
        base64エンコードされたwavデータを一纏めにし、wavファイルで返します。
        """
```